### DISCUSSION TITLE
I am getting a weird unknown error while processing a .mp4 file

### DISCUSSION THREAD

**Discussion by rushabtated4** [deepgram/community#317] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

Hi, while processing a mp4 file, it just shows pending for sometime and then the status changes to unknown and the call fails
<img width="574" alt="Screenshot 2023-08-15 at 10 40 01 AM" src="https://github.com/deepgram/community/assets/142289854/e05c0194-19dd-4474-9538-c3a0a7230a49">
<img width="595" alt="Screenshot 2023-08-15 at 10 39 53 AM" src="https://github.com/deepgram/community/assets/142289854/dfdc4e94-5f21-499d-aa69-ee146417de28">

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

POST /v1/listen?model=general&tier=base&detect_language=true&punctuate=true&diarize=true&smart_format=true&callback=https%3A%2F%2Fpodcastsapp.bubbleapps.io%2Fapi%2F1.1%2Fwf%2Fdeepgram&paragraphs=true&detect_topics=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

ab7c058b-b6d4-4578-aeb8-26d5dd6d6290

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @rushabtated4, thanks for reporting this issue, and I'm sorry it's causing you trouble. This is a known issue that affects a small number of requests, and we are investigating a root cause. I recommend retrying these requests that do not return a response in a timely manner.
